src/main/target/UAVPNG030MINI/target.mk

```diff
@@ -0,0 +1,6 @@
+F405_TARGETS   += $(TARGET)
```

Comment:
I'm interested in knowing if the current code base works for F407, as this board is the first instance of F407, even though F405 and F407 are supposed to be software compatible.

Reply:
I think there might be subtle differences like the interrupt vector, and I also think someone used an F407 discovery/nucleo board in our slack, maybe it was @chrish?

Reply:
The maidenflight was successful done using Betaflight 3.3.0 when we wrote that target file.
So it essentially works...

Reply:
@DieHertz Ah yeah, STM32F4DISCOVERY target is the one.
@aguindehi It's good to hear the successful maiden because I assume the discovery board has never flown ;)

Reply:
We started the port from the LPC2148 to STM32 on the discovery board back in 2013 when we made NGOS support multiple architectures. I think we never got it airborne on the discovery board but we did all prelimary changes on that board until we had real hardware supporting the STM architecture. I just checked, the discovery startup code still exists in NGOS. ;)